src/originalBudgetTrackingApp/import/importTransactions.ts

```diff
@@ -120,9 +121,3 @@ function transactionsDateComparator(t1: Transaction, t2: Transaction) {
   }
   return 1;
 }
-
-export function calculateTransactionHash({
-  date, chargedAmount, description, memo
-}: Transaction, companyId: string, accountNumber: string) {
-  return `${date}_${chargedAmount}_${description}_${memo}_${companyId}_${accountNumber}`;
-}
```

Comment:
I moved this function (and maybe more functions in the future) to a separate file called `transactions/transactions`, this is something like "*Transactions Utils*"- helper methods to work with transactions (because I needed this function also in `JsonExporter`).

In addition, I removed the `companyId` and `accountNumber` from the hash, because we need the hash to easily identify transactions **no matter they come from**, and if we want to merge the imported transactions with existed transactions in the output vendor, we need to recalculate the hash for transactions in the output.
In short, I think we need to calculate the hash from the `Transaction`/`EnrichrdTransaction` itself. (So maybe add  `companyId` and `accountNumber` to the interface.

@brafdlog WDYT?

Reply:
OK, sounds right.